app_dart/lib/src/request_handlers/update_existing_flaky_issues.dart

```diff
@@ -24,27 +24,30 @@ import 'flaky_handler_utils.dart';
 /// The query parameter kThresholdKey is required in order for the handler to
 /// properly adjusts the priority labels.
 @immutable
-class UpdateExistingFlakyIssue extends ApiRequestHandler<Body> {
+final class UpdateExistingFlakyIssue extends ApiRequestHandler<Body> {
   const UpdateExistingFlakyIssue({
     required super.config,
     required super.authenticationProvider,
-    @visibleForTesting this.ciYaml,
-  });
+    required BigQueryService bigQuery,
+    @visibleForTesting this.ciYamlForTesting,
+  }) : _bigQuery = bigQuery;
 
   static const String kThresholdKey = 'threshold';
   static const int kFreshPeriodForOpenFlake = 7; // days
 
-  final CiYamlSet? ciYaml;
+  final BigQueryService _bigQuery;
+
+  // TODO(matanlurey): Use `CiYamlFetcher` instead.
+  final CiYamlSet? ciYamlForTesting;
```

Comment:
Did you mean for this to also be `@visibleForTesting` since it is non-private?

Reply:
I'll just remove it in the next PR :)